figure.html

```diff
@@ -12,7 +12,7 @@
 
 <p>The majority of OMERO servers will be configured so that OMERO.figure is accessible using a tab in the OMERO.web client, although in some cases OMERO.figure might have a unique URL.</p>
 
-<p>The main features of OMERO.figure are also illustrated in short <a href="http://will-moore.github.io/figure/videos.html" target="_blank">videos</a> on the OMERO.figure web site.<p>
+<p>The main features of OMERO.figure are also illustrated in short <a href="http://figure.openmicroscopy.org/videos.html" target="_blank">videos</a> on the OMERO.figure web site.<p>
```

Comment:
Not this PR's fault, but shouldn't this line end with a `</p>`? (Could Travis do some kind of weblint on these pages?)